caas/kubernetes/provider/bootstrap.go

```diff
@@ -538,7 +539,7 @@ func (c *controllerStack) getControllerSvcSpec(cloudType string, cfg *podcfg.Boo
 	return spec, nil
 }
 
-func (c *controllerStack) createControllerProxy() error {
+func (c *controllerStack) createControllerProxy(ctx context.Context) error {
```

Comment:
Just checking there is logic to cleanup these? I'm guessing the namespace delete is sufficient but just checking it doesn't hold up the controller destroy?

Reply:
They will get cleaned up on namespace deletion so I have left it out. Can add it?